t/04-check_vars_docu.t

```diff
@@ -70,17 +73,22 @@ sub read_doc {
             elsif ($reading) {
                 next if ($line =~ /$table_header/);
                 my ($var, $value, $default, $explanation) = $line =~ /^([^;]+);\s*([^;]*);\s*([^;]*);\s*(.*)$/;
+
                 next unless ($var);
                 $default = '' unless (defined $default);
                 $value   = '' unless (defined $value);
                 unless ($explanation) {
-                    say "still missing explanation for backend $backend variable $var";
+                    say "" if ($errors == 0);
+                    say color ('red') . "Error: still missing explanation for backend $backend variable $var";
+                    $errors++;
```

Comment:
The current output is:
```
04-check_vars_docu.t ................... 
Dubious, test returned 1 (wstat 256, 0x100)
```
This usually means that a test was aborted unexpectedly.
I think it would be better to do
`fail "Error: still missing explanation for backend $backend variable $var"`

Also a note about `color()`: this will add the escape sequence for the color, but you never add the escape sequence for ending the color, so all output after that would be colored too.
It's better to use `colored()`.

Reply:
I would also prefer the `fail` line suggested above, with no extra color. That would create failures in the test plan, which is what we want.

Reply:
Done